e2e/test_plugin_installations.py

```diff
@@ -95,7 +94,7 @@ def test_sample_plugins_installed(self):
             "workflow-step-api": "2.14",
             "workflow-support": "2.17"
         }
-        for plugin, version in installed_plugin_versions.iteritems():
+        for plugin, version in expected_plugin_versions.iteritems():
```

Comment:
So, as I understand we want to just make sure the plugins we care about are installed, instead of explicitly tagging every single plugin version?

This is good with me, but I think it makes sense then to shrink this expected plugin list, since many of the plugins are dependencies that we don't necessarily care about. I think I'd be in favor to just making this equivalent to our test_data/plugins.yml plugins.

Reply:
I agree. Do you mind if I do that in a separate PR in order to get this over the line?

Reply:
go for it